zebra/interface.c

```diff
@@ -59,6 +59,9 @@ static void if_zebra_speed_update(struct event *thread)
 	bool changed = false;
 	int error = 0;
 
+	if (!ifp->vrf || ifp->vrf->vrf_id == VRF_UNKNOWN)
+		return;
```

Comment:
What's the code path that the speed is set in this case?

Reply:
actually, with recent version, there is no need to do this check.
I removed the commit from the pull request and rebased.